.github/actions/support-package-metadata-json/metadata.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import json
+import os
+from datetime import datetime
+
+
+def main():
+    support_pkg_dir = os.environ['SUPPORT_PACKAGE_TMP_DIR']
+
+    for mod_ver in os.listdir(support_pkg_dir):
+        version_dir = os.path.join(support_pkg_dir, mod_ver)
+
+        with open(os.path.join(version_dir, 'latest'), 'r') as f:
+            latest_file = f.read().splitlines()[0]
```

Comment:
[nit] 
```suggestion
            latest_file = f.read().readline()
```

Reply:
Strings don't have a `readline()` method, I had this in a [previous commit](https://github.com/stackrox/collector/pull/981/commits/1aa3b1f9f1b1ee1e399b218cdc7566ad535410ae):
```py
latest_file = f.readline()
```

Problem is `readline()` from a file keeps the line end character, so I would still need to remove that. Splitting the line and keeping the first part seemed the cleanest and most reliable method, WDYT?

Reply:
It doesn't very important. I meant to write `latest_file = f.readline()`. You could try `latest_file = f.readline().strip()`. It does avoid reading the entire file. In this case it is not important, but in some other case it might be important if the file is large.

Reply:
Alright, `f.readline().strip()` is easier to read, I'll give it a try.